libs/utils/src/vec_map.rs

```diff
@@ -9,6 +9,8 @@ pub enum VecMapOrdering {
 /// Ordered map datastructure implemented in a Vec.
 /// Append only - can only add keys that are larger than the
 /// current max key.
+/// Ordering can be adjusted using [VecMapOrdering]
```

Comment:
I think you need to do [`VecMapOrdering`] for the docs to be properly generated. You can check by running `cargo doc --workspace --no-deps --document-private-items`.

Reply:
I think both variations are correct, I did run the command before pushing

Reply:
I found the example here https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html#linking-to-items-by-name. But I can add ticks to adapt to the rest of the codebase, if that is the case?

Reply:
Ah neat. Wasn't aware both work. We generally use the backticks throughout the codebase.